week-3/extended-project/nowhere/views/index.erb

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Nowhere
+    </title>
+    <!-- Meta tag Keywords -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta name="keywords" content="Hi Codeable!" />
+    <script>

```

Comment:
You should never ever add JavaScript to the `head` of a website, always on the `body` at the bottom.